### ISSUE TITLE
Improve accessibility of Website

### ISSUE THREAD

**Issue by sachsom95** [MLH-Fellowship/portfolio-template#62]
The website has minor accessibility issues such as missing [alt] tag for images and names for links making it difficult for website readers. 

- [ ] alt images
- [ ]  links -> name
- [ ] check if the website is color blind friendly 

Any other suggestions?

**Comment by sachsom95** [comment]
You can test basic accessibility issues using chrome lighthouse

**Comment by alexzender45** [comment]
Please assign this issue to my team

**Comment by alexzender45** [comment]
> The website has minor accessibility issues such as missing [alt] tag for images and names for links making it difficult for website readers.
> 
> * [ ]  alt images
> * [ ]   link have names
> * [x ]  check if the website is color blind friendly
> 
> Any other suggestions?
